packages/docs/docs/guides/7-request storage subgraph/0-entities-storage.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Storage Subgraph Entities
+keywords: [Subgraph, Entities, Schema]
+description: Subgraph entities and descriptions.
+---
```

Comment:
missing an introduction about the storage, link to repo. plus an explanation that it's not deployed on TheGraph network, but on a private node. I think we have documentation on how to run it in the [repo](https://github.com/RequestNetwork/storage-subgraph)

Reply:
Agreed.